scripts/drone/steps/lib.star

```diff
@@ -356,6 +356,35 @@ def e2e_tests_artifacts():
         ],
     }
 
+def playwright_e2e_report_upload():
+    return {
+        "name": "playwright-e2e-report-upload",
+        "image": images["cloudsdk"],
+        "depends_on": [
+            "playwright-plugin-e2e",
+        ],
+        "failure": "ignore",
+        "when": {
+            "status": [
+                "success",
+                "failure",
+            ],
+        },
+        "environment": {
+            "GCP_GRAFANA_UPLOAD_ARTIFACTS_KEY": from_secret(gcp_upload_artifacts_key),
+            "GITHUB_TOKEN": from_secret("github_token"),
+        },
```

Comment:
Will these steps run on every PR? Is there a risk of someone else opening a PR where they change the build script and just echo the secrets in the console so it get exposed?

Reply:
As discussed offline, the Drone build does not run if the changes in drone.yaml aren't signed, and only org members have access to the key used for signing.